spinalcordtoolbox/deepseg/models.py

```diff
@@ -256,7 +259,16 @@ def install_model(name_model):
     :return: None
     """
     logger.info("\nINSTALLING MODEL: {}".format(name_model))
-    download.install_data(MODELS[name_model]['url'], folder(name_model))
+    url_field = MODELS[name_model]['url']
+    # List of mirror URLs corresponding to a single model
+    if isinstance(url_field, list):
+        model_urls = url_field
+        download.install_data(model_urls, folder(name_model))
+    # Dict of lists, with each list corresponding to a different model seed for ensembling
+    elif isinstance(url_field, dict):
+        for seed_name, model_urls in url_field.items():
+            logger.info(f"\nInstalling '{seed_name}'...")
+            download.install_data(model_urls, folder(os.path.join(name_model, seed_name)), keep=True)
```

Comment:
NB: Changing from `folder(name_model)` to `folder(os.path.join(name_model, seed_name))` is what implements the folder structure proposed in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/4036#issue-1579823819:

![image](https://user-images.githubusercontent.com/16181459/219148121-88c9a0f2-b385-4215-8e7c-64e8f1efcc0c.png)

(The last subfolders (the additional `model_seg_lesion_mp2rage` folders) are contained in the model archives themselves, I believe.)